content/momentum/changelog/4/4-5-0.md

```diff
@@ -0,0 +1,70 @@
+---
+lastUpdated: "11/15/2022"
+title: "Momentum 4.5.0 Changelog"
+description: "Momentum 4.5.0 was released on 2022-12-01. This section will list all of the major changes that happened with the release of Momentum 4.5.0 Depending on installation type, all changes may not be applicable"
+---
+
+**Momentum 4.5.0** was released on 2022-12-01. This section will list all of the major changes that happened with the release of **Momentum 4.5.0**. Depending on installation type, all changes may not be applicable
+
+<a name="changelog.4.5.0.table"></a>
+
+| Type | Ticket | Description |
+| --- | --- | --- |
+| Fix | SD-2935 | Crash when outbound_throttle_messages is configured at the same time as core_mailq_throttle_message hook |
+| Fix | SD-2995 | soft_bounce functionality periodically tries using an invalid connection handle causing a crash |
+| Fix | SD-3167 | Stack overflow in modules/delivery/http2/http2clnt.c http_send |
+| Fix | SD-3259 | Momentum incorrectly interprets checkpoint file data on restart |
+| Fix | SD-3263 | DKIM Oversigning to Avoid Replay Attacks |
+| Fix | SD-3288 | Transmission API does not sanity check recipient list for empty array |
+| Fix | SD-3335 | lu_pull not working on AL2 nodes |
+| Fix | SD-3374 | lu_pull not working on AL2 nodes, round 2 |
```

Comment:
These should be combined and reworded- "Files created from lu_pull are now all owned by ecuser"

Reply:
Fixing!